controllers/vsl.go

```diff
@@ -11,6 +12,16 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+// provider specific object storage
+const (
+	Profile          = "profile"
```

Comment:
For these, also include for which cloud provider the constant is for.

something like:
```
AwsProfile
AzureSubscriptionId
```
```